.github/workflows/coding-standards.yml

```diff
@@ -50,6 +53,67 @@ jobs:
         with:
           composer-options: "--no-suggest"
 
+      - name: "Format the code"
+        continue-on-error: true
+        run: |
+          mkdir .cache
+          ./vendor/bin/phpcbf
+
```

Comment:
@GromNaN what are your thoughts on this? I'm usually not a fan of workflows that commit changes, but since it's "just" checkstyle it may be fine.